src/bridge/cockpitpeer.c

```diff
@@ -183,7 +200,13 @@ on_other_closed (CockpitTransport *transport,
   CockpitPipe *pipe;
   gint status = 0;
 
-  if (self->inited)
+  if (!self->inited)
+    {
+      g_debug ("%s: bridge failed to start%s%s", self->name,
```

Comment:
Should this say ".. before inited", to make it clearer that it is in this situation?